spec/models/form_profile_spec.rb

```diff
@@ -916,7 +916,7 @@ def self.test_pciu_us_phone(primary, expected)
 
   describe '#prefill_form' do
     def can_prefill_emis(yes)
```

Comment:
Does `#can_prefill_emis` need to be renamed?

Reply:
yes good catch!

Reply:
yes! thank you! 🦅  👀

Reply:
Decided to not rename this since the method is used so much in this file and this file is being modified in another upcoming PR.